polar_route/crossing.py

```diff
@@ -85,34 +85,35 @@ def __init__(self,source_graph=None,neighbour_graph=None,
         self.debugging     = debugging
 
     def _newton_optimisation(self, f, x, a, Y, u1, v1, u2, v2, s1, s2):
-        """
-            Find the y value of the crossing point and travel time in each cell using the Newton method
-        """
+        '''
+            FILL
+        '''
         y0 = (Y*x)/(x+a)
         if self.debugging:
             print('---Initial y={:.2f}'.format(y0))
         improving = True
-        iteration_num = 0
+        iterartion_num = 0
```

Comment:
Could you keep this variable name as `iteration_num` (or something similar)? "iterartion" is a typo I fixed in a previous commit

Reply:
Thanks for the spot I have added this to the next commit